### PR TITLE
[Button.Group] add dynamic update methods

### PR DESCRIPTION
**Pull request by CApopsicle**
add two methods for `<Button.Group>` 
- `setOptions([newOptions])` 
- `setSelectedOption(optionValue)` 

@Dsweih

### CODE REVIEW

**Comment by desmond9319** [comment]
test webhook

**Comment by CApopsicle** [comment]
test webhook

File: src/components/button/index.js [hunk 1]
```diff
@@ -84,6 +84,17 @@ Button.Group = {
       this.buttonOptions.forEach((buttonOption) => this.$set(buttonOption, 'selected', false))
       this.$set(this.buttonOptions[idx], 'selected', true)
     }),
+    setOptions(newOpts) {
+      this.buttonOptions = newOpts
+      this.selectedIdx = newOpts.find((option) => option.selected === true)
```
Comment (desmond9319) [inline src/components/button/index.js]: 可以不用 `=== true` ?

Comment (CApopsicle) [reply src/components/button/index.js]: fixed!

**Review by desmond9319 — APPROVED** [review]

**Comment by desmond9319** [comment]
test webhook

**Comment by desmond9319** [comment]
test webhook again

**Pull request merged**